Koronavilkku/Koronavilkku/Extension/UIView.swift

```diff
@@ -58,16 +58,46 @@ extension UIView {
         return view.snp.bottom
     }
     
-    static func createDivider() -> UIView {
+    func appendView(_ view: UIView, insets: UIEdgeInsets, top: ConstraintItem) -> ConstraintItem {
+        addSubview(view)
+        
+        view.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview().inset(insets)
+            make.top.equalTo(top).offset(insets.top)
+        }
+        
+        return view.snp.bottom
+    }
+    
+    static func createDivider(height: CGFloat = 0.5) -> UIView {
         let divider = UIView()
         divider.backgroundColor = UIColor.Greyscale.borderGrey
         
         divider.snp.makeConstraints { make in
-            make.height.equalTo(0.5)
+            make.height.equalTo(height)
         }
         
         return divider
     }
+    
+    @discardableResult
+    func layout(appendMaker: ((UIView, UIEdgeInsets?) -> ()) -> ()) -> Self {
```

Comment:
Could we use typealias here to be able to provide default value for optional parameter